packages/devtools_app/test/cpu_profiler/cpu_profiler_controller_test.dart

```diff
@@ -25,6 +26,14 @@ void main() {
   group('CpuProfileController', () {
     late CpuProfilerController controller;
 
+    Future<void> disableAllFiltering() async {
+      for (final filter in controller.activeFilter.value.toggleFilters) {
+        filter.enabled.value = false;
+      }
+      controller.setActiveFilter();
+      await shortDelay();
```

Comment:
What's the reason for the short delay? Can you add a comment?

Reply:
added comment